master/internal/grpcutil/auth.go

```diff
@@ -40,7 +40,7 @@ var (
 	// ErrInvalidCredentials notifies that the provided credentials are invalid or missing.
 	ErrInvalidCredentials = status.Error(codes.Unauthenticated, "invalid credentials")
 	// ErrTokenMissing notifies that the bearer token could not be found.
-	ErrTokenMissing = status.Error(codes.InvalidArgument, "token missing")
+	ErrTokenMissing = status.Error(codes.Unauthenticated, "token missing")
```

Comment:
The `InvalidArgument` code was very confusing to me when debugging

Reply:
Note to check: this feels like the kinda thing to be careful about (e.g. make sure the CLI doesn't rely on a 400 in a bad way somewhere).

Reply:
I searched the codebase for "400", "bad request" and "badrequest". The only relevant thing I saw was that the CLI will handle displaying the error slightly differently (displaying a traceback instead of printing the Exception) - https://github.com/determined-ai/determined/blob/master/harness/determined/cli/cli.py#L283.

But since that is how we currently handle unauthenticated errors, I think it's fine? Or we could add Unauthenticated errors to the list of errors where we print out the Python Exception

Reply:
Yeah, looks fine to me.